server/src/main/java/org/opensearch/node/GlobalPerformanceStats.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.node;
+
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+import org.opensearch.core.xcontent.ToXContentFragment;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class represents performance stats such as CPU, Memory and IO resource usage of each node along with the time
+ * elapsed from when the stats were recorded.
+ */
+public class GlobalPerformanceStats implements Writeable, ToXContentFragment {
```

Comment:
We have "NodePerformanceStatistics" which stores local node stats - so got a comment that 'GlobalPerformanceStats' works here since we capture local + other nodes of the cluster. 

In actual stats which gets exposed to the customer - I have kept only "performance_stats" ,